closure/goog/math/integer.js

```diff
@@ -589,6 +643,13 @@ goog.math.Integer.prototype.divide = function(other) {
     return this.divide(other.negate()).negate();
   }
 
+  // Have to degrade to slowDivide for Very Large Numbers, because
+  // they're out of range for the floating-point approximation
+  // technique used below.
+  if(this.bits_.length > 30) {
```

Comment:
On a side note, running your PRs through gjslint would probably be helpful: https://github.com/google/closure-linter